public/index.html

```diff
@@ -1,79 +0,0 @@
-  <meta name="author" content="getGithub">
-  <meta name="description" content="getGithub">
-  <meta name="viewport" content="initial-scale=1">
-  <link rel="stylesheet" href="css/style.css">
-  <title>getGithub</title>
-</head>
-  <body>
-    <header>
-      <nav>
-        <div class="container">
-          <form id="searchForm" method="post">
-            <input type="text" name="username" value="" placeholder="Enter username..." required maxlength="20">
-            <input type="submit" name="" value="submit">
-          </form>
-        </div>
-      </nav>
-    </header>
-    <div class="container main">
-      <aside class="profileInfo">
-        <!-- profile info -->
-        <div class="avatar">
-          <img src="https://avatars3.githubusercontent.com/u/14959?v=4&s=460" alt="user avatar">
-        </div>
-        <div class="names">
-          <h1>Name</h1>
-          <h6>username</h6>
-        </div>
-        <div class="address">
-          <p id="company">Company</p>
-          <p id="location">location</p>
-        </div>
-      </aside>
-      <div class="repos">
-        <div class="status">
-          <p id="followersNumber">Followers <span>2435</span></p>
-          <p id="followingNumber">Following <span>542</span></p>
-        </div>
-        <div class="listRepos">
-          <div class="repoData">
-            <!-- repos data ,, this will be duplicated-->
-            <a href="#" class="repoName">my repor name</a>
-            <p class="repoDesc">my description fkdsjflkjkldsj</p>
-            <p class="repoLang">Language : <span>Jave-script</span></p>
-          </div>
-          <div class="repoData">
-            <!-- repos data ,, this will be duplicated-->
-            <a href="#" class="repoName">my repor name</a>
-            <p class="repoDesc">my description fkdsjflkjkldsj</p>
-            <p class="repoLang">Language : <span>Jave-script</span></p>
-          </div>
-          <div class="repoData">
-            <!-- repos data ,, this will be duplicated-->
-            <a href="#" class="repoName">my repor name</a>
-            <p class="repoDesc">my description fkdsjflkjkldsj</p>
-            <p class="repoLang">Language : <span>Jave-script</span></p>
-          </div>
-          <div class="repoData">
-            <!-- repos data ,, this will be duplicated-->
-            <a href="#" class="repoName">my repor name</a>
-            <p class="repoDesc">my description fkdsjflkjkldsj</p>
-            <p class="repoLang">Language : <span>Jave-script</span></p>
-          </div>
-          <div class="repoData">
-            <!-- repos data ,, this will be duplicated-->
-            <a href="#" class="repoName">my repor name</a>
-            <p class="repoDesc">my description fkdsjflkjkldsj</p>
-            <p class="repoLang">Language : <span>Jave-script</span></p>
-          </div>
-        </div>
-      </div>
-    </div>
-    <script type="text/javascript" src="js/index.js"></script>
-  </body>
-</html>
```

Comment:
All the fake data here should be removed. Just a welcome message and some instructions should be enough.